homeassistant/components/light/services.yaml

```diff
@@ -2,28 +2,44 @@
 
 turn_on:
   description: Turn a light on.
+  target:
+    name: Target
+    description: What to turn on
+    example: "light.kitchen"
+    selector:
+      target:
+        entity:
+          domain: light
```

Comment:
I think that we can leave this out. If no selector given, default to this selector with same domain as the service.